OmiseSDK/SourcePaymentMethod.swift

```diff
@@ -37,6 +37,57 @@ public enum PaymentInformation: Codable, Equatable {
 
     /// Online Alipay + Hongkong Wallet Payment Source
     case alipayHK
+    
+    /// The Atome customer information
+    public struct Atome: PaymentMethod {
+
+        public static var paymentMethodTypePrefix: String = OMSSourceTypeValue.atome.rawValue
+
+        public var type: String = OMSSourceTypeValue.atome.rawValue
+
+        /// The customers phone number. Contains only digits and has 10 or 11 characters
+        public let phoneNumber: String
+        public let name: String?
+        public let email: String?
+        public let shippingStreet: String?
+        public let shippingCity: String?
+        public let shippingCountryCode: String?
+        public let shippingPostalCode: String?
+
+        private enum CodingKeys: String, CodingKey {
+            case name
+            case email
+            case phoneNumber = "phone_number"
+            case shippingStreet = "shipping_street"
+            case shippingCity = "shipping_city"
+            case shippingCountryCode = "shipping_country_code"
+            case shippingPostalCode = "shipping_postal_code"
+        }
+
+        /// Creates a new Atome source with the given customer information
+        ///
+        /// - Parameters:
+        ///   - phoneNumber:  The customers phone number
+        public init(phoneNumber: String,
+                name: String? = nil,
+                email: String? = nil,
+                shippingStreet: String? = nil,
+                shippingCity: String? = nil,
+                shippingCountryCode: String? = nil,
+                shippingPostalCode: String? = nil) {
```

Comment:
I believe the shipping postal code, country code, and street are also mandatory.

Reply:
Updated require fields as https://github.com/omise/omise-ios/pull/184/commits/bc0a068b875d28319dd0240abad5e07221c1b291 ka. 🙏